department-of-veterans-affairs/abd-vro
# Update Contention Classification Lookup Table to v0.2

## What was the problem?

We want more diagnostic codes mapped.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/177

## How does this fix it?[^1]

- follows instructions in documentation on how to update table versions ([HERE](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Updating-Contention-Classification-DC-Lookup-Table))

## How to test this PR
- `pytest` command for unit tests
- ensure that the `table_version.py` matches desired version


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
